tests/test_sdk/conftest.py

```
@@ -0,0 +1,162 @@
+                ('Bob Jones', 'bob@example.com', 'Los Angeles'),
+                ('Carol White', 'carol@example.com', 'New York')
+            ON CONFLICT (email) DO NOTHING;
+
+            INSERT INTO orders (customer_id, product, amount, order_date) VALUES
+                (1, 'Widget', 29.99, '2024-01-15'),
+                (1, 'Gadget', 49.99, '2024-01-20'),
+                (2, 'Widget', 29.99, '2024-02-01');
+        """)
+        conn.commit()
+        conn.close()
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)